qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -1356,6 +1356,10 @@ def url(self, *, requested=False):
         else:
             return page.url()
 
+    def pid(self) -> int:
```

Comment:
There is `renderer_process_pid` already.

Reply:
did not see it, thank you.